src/QY2Styler.h

```diff
@@ -167,6 +167,25 @@ class QY2Styler : public QObject
      **/
     void processUrls( QString & text );
 
+    /**
+     * Build a stylesheet from a string.
+     */
+    const QString buildStyleSheet(QString content);
+
+    /**
+     * Build a stylesheet from a string.
+     *
+     * Receives a list of already imported files.
+     */
+    const QString buildStyleSheet(QString content, QStringList & alreadyImported);
```

Comment:
It is not apparent whether `alreadyImported` are filenames or the contents of the files. `alreadyImportedFilenames`, `alreadyImportedContents`?